Vision/README.md

```diff
@@ -11,34 +9,34 @@ Our vision is for a World Wide Web that is more inclusive, and more respectful o
 
 We will improve the fundamental integrity of the Web platform. The Web will not only grow in scope and importance in our lives; it will grow in respect for its users, grow in the trust of its users, grow in its inclusion of all humanity as its users.
 
-Our new vision is built on these twin pillars:
-
-* Our existing pillar, on which we built the web: the enormous potential of technological change.
-* Our new pillar for the years ahead: recognition and embodiment of fundamental values into the architecture of the web.
```

Comment:
I removed these "pillars" - because I think they're not important as is. The "recognition and embodiment of fundamental values" is critical - and it is covered in the Mission statement.

Reply:
replicating comments from https://github.com/w3c/AB-public/pull/28#discussion_r1048815862:

(thumbs-up from mchampion)

@[dwsinger](https://github.com/dwsinger) dwsinger [1 hour ago](https://github.com/w3c/AB-public/pull/28#discussion_r1049046656)
I'm concerned that we then say nothing about our continued belief about our 'old value', the potential of the web to change the world.